samples/estimation/df-chemistry/src/df_chemistry.qs

```diff
@@ -20,28 +20,28 @@ namespace Microsoft.Quantum.Applications.Chemistry {
     /// # Reference
     /// arXiv:2007.14460, p. 9, eq. 9
     newtype DoubleFactorizedChemistryProblem = (
-        // Number of orbitals (N, p. 8)
-        NumOrbitals: Int,
-        // one-body norm (ǁL⁽⁻¹⁾ǁ, p. 8, eq. 16)
-        OneBodyNorm: Double,
-        // one-body norm (¼∑ǁL⁽ʳ⁾ǁ², p. 8, eq. 16)
-        TwoBodyNorm: Double,
-        // eigenvalues in the EVD of the one-electron Hamiltonian (λₖ, p. 54, eq. 67)
-        OneBodyEigenValues: Double[],
-        // eigenvectors in the EVD of the one-electron Hamiltonian (Rₖ, p. 54, eq. 67)
-        OneBodyEigenVectors: Double[][],
-        // norms inside Λ_SH (p. 56, eq. 77)
-        Lambdas: Double[],
-        // eigenvalues in the EVDs of the two-electron Hamiltonian for all r (λₖ⁽ʳ⁾, p. 56, eq. 77)
-        TwoBodyEigenValues: Double[][],
-        // eigenvectors in the EVDs of the two-electron Hamiltonian for all r (R⁽ʳ⁾ₖ, p. 56, eq. 77)
-        TwoBodyEigenVectors: Double[][][],
+    // Number of orbitals (N, p. 8)
```

Comment:
Again, should these be indented a level relative to line 22?

Reply:
Indentation logic will be addressed by a future PR.